pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -27,16 +27,31 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 	"kubevirt.io/client-go/log"
 
-	"kubevirt.io/kubevirt/pkg/network/cache"
-	netsetup "kubevirt.io/kubevirt/pkg/network/setup"
 	netvmispec "kubevirt.io/kubevirt/pkg/network/vmispec"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/cli"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/converter"
 )
 
-func hotplugVirtioInterface(vmi *v1.VirtualMachineInstance, converterContext *converter.ConverterContext, dom cli.VirDomain, domain *api.Domain) error {
-	vmConfigurator := netsetup.NewVMNetworkConfigurator(vmi, cache.CacheCreator{})
+type vmConfigurator interface {
+	SetupPodNetworkPhase2(domain *api.Domain, networksToPlug []v1.Network) error
+}
+
+type VirtIOInterfaceManager struct {
+	converterContext *converter.ConverterContext
+	dom              cli.VirDomain
+	configurator     vmConfigurator
+}
+
+func newVirtIOInterfaceManager(ctx *converter.ConverterContext, libvirtClient cli.VirDomain, configurator vmConfigurator) *VirtIOInterfaceManager {
```

Comment:
nit: Please consider renaming the `ctx` parameter.
The parameter name plus it being the first parameter, resembles Go's convention of using `ctx context.Context` as the first parameter of a function.
https://pkg.go.dev/context#pkg-overview

Reply:
Done.